packages/discovery-react-components/src/components/SearchFacets/SearchFacets.tsx

```diff
@@ -187,7 +195,8 @@ const SearchFacets: FC<SearchFacetsProps> = ({
           <FieldFacets
             allFacets={allFieldFacets}
             showMatchingResults={showMatchingResults}
-            onChange={handleOnChange}
+            onChange={onChange}
+            onSearchFacetsChange={handleOnChange}
```

Comment:
I'd recommend changing the name of `handleOnChange` to `handleOnSearchFacetsChange`, since I think that matches how the existing `onChange` has been renamed. Maybe we should even make the new `onChange` clearer in its naming, since we have two similar functions.